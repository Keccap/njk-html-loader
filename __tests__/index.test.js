const compiler = require('./compiler');

describe('index', () => {
  test('should generate correct code', async () => {
    const stats = await compiler('fixtures/index.njk', { });
    const output = stats.toJson().modules[0].source;

    expect(output).toMatchSnapshot();
  });
});
