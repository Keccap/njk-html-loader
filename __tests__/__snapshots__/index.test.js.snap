// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should generate correct code 1`] = `"module.exports = \\"<!DOCTYPE html>\\\\r\\\\n<html lang=\\\\\\"ru\\\\\\">\\\\r\\\\n<head>\\\\r\\\\n  <meta charset=\\\\\\"utf-8\\\\\\">\\\\r\\\\n  <meta name=\\\\\\"viewport\\\\\\" content=\\\\\\"width=device-width, initial-scale=1, maximum-scale=1\\\\\\">\\\\r\\\\n  <meta http-equiv=\\\\\\"X-UA-Compatible\\\\\\" content=\\\\\\"IE=edge\\\\\\">\\\\r\\\\n  <meta name=\\\\\\"format-detection\\\\\\" content=\\\\\\"telephone=no\\\\\\">\\\\r\\\\n  <meta name=\\\\\\"format-detection\\\\\\" content=\\\\\\"address=no\\\\\\">\\\\r\\\\n  <title>Simple</title>\\\\r\\\\n</head>\\\\r\\\\n<body>\\\\r\\\\n  Test template\\\\r\\\\n</body>\\\\r\\\\n</html>\\\\r\\\\n\\";"`;
